extensions/api/data-management/contractdefinition/src/main/java/org/eclipse/dataspaceconnector/api/datamanagement/contractdefinition/transform/ContractDefinitionDtoToContractDefinitionTransformer.java

```diff
@@ -39,6 +39,7 @@ public Class<ContractDefinition> getOutputType() {
         return ContractDefinition.Builder.newInstance()
                 .id(object.getId())
                 .accessPolicyId(object.getAccessPolicyId())
+                .createdAt(object.getCreatedAt())
```

Comment:
separating the DTO's sounds like something for another PR.

Reply:
@paullatzelsperger yes, I thought the same was for `PolicyDefinition` and `AssetDto` but the first doesn't have a DTO (should it?) and on the second the `createdAt` information is not present. For consistency I think it should be on every entity-dto.

Reply:
the `AssetDto`'s field should be added, I overlooked that. 

Not sure a `PolicyDefinitionDto` will add much value, because it will likely only carry an ID and the `Policy`, which is already intended to be serializable. Likewise, introducing a DTO for `Policy` will only cause more problems with extensibility, without adding any value.

Reply:
@paullatzelsperger I wouldn't add a DTO for the `Policy`, but I'd for `PolicyDefinition`, so we could separate the input from the output one, as we'd do for `ContractDefinition`

Reply:
but, yeah, let's do all these chores in another PR

Reply:
actually, I noticed that other DTOs don't have the `createdAt` flag either, so I'll add it there too.

Reply:
@ndr-brt I added the `createdAt` and `updatedAt` field to the DTOs in the form of a `BaseDto` and `MutableDto` respectively